featuretools/feature_base/feature_base.py

```diff
@@ -328,21 +328,23 @@ def unique_name(self):
 class IdentityFeature(FeatureBase):
     """Feature for entity that is equivalent to underlying variable"""
 
-    def __init__(self, variable):
+    def __init__(self, variable, name=None):
         entity_id = variable.entity_id
         self.variable = variable.entityset.metadata[entity_id][variable.id]
         self.return_type = type(variable)
         super(IdentityFeature, self).__init__(entity=variable.entity,
                                               base_features=[],
                                               relationship_path=[],
                                               primitive=PrimitiveBase)
+        if name is not None:
+            self._name = name
 
     @classmethod
     def from_dictionary(cls, arguments, entityset, dependencies, primitives_deserializer):
         entity_id = arguments['entity_id']
         variable_id = arguments['variable_id']
         variable = entityset[entity_id][variable_id]
-        return cls(variable)
+        return cls(variable, arguments['name'])
```

Comment:
Let's not rely on a positional call.
Lets do 
`return cls(variable=variable, name=arguments['name'])` so we specify what the arguments are.